<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OWTF Valhalla</string>
  </property>
  <widget class="QWidget" name="mainWindowCentralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabExecute">
       <attribute name="title">
        <string>Execute</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabExecuteCerticalLayout"/>
      </widget>
      <widget class="QWidget" name="tabManageContainers">
       <attribute name="title">
        <string>Manage Containers</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabManageVerticalLayout"/>
      </widget>
      <widget class="QWidget" name="tabResults">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabResultsVerticalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1184</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
